### ISSUE TITLE
Returning blank file

### ISSUE THREAD

**Issue by guicunha** [rap2hpoutre/fast-excel#130]
When i passing a collection like it:

`return fastexcel($debitos)->download('file.xlsx');`

or like it:

`return (new FastExcel($debitos))->download('file.xlsx');`

the file comes with no data

**Comment by rjsandim** [comment]
Did you solve this error? I'm facing the same.

**Comment by rjsandim** [comment]
The problem here was my NGINX server. 

When I was using download method for large files...NGINX didn’t work properly.

So I had to change 2 variables:

fastcgi_buffers 1024 1024k; 
fastcgi_buffer_size 1024k;

**Comment by rap2hpoutre** [comment]
@rjsandim Thank you for your suggestion. @guicunha Did it solve your problem?

**Comment by areynolds04** [comment]
@rap2hpoutre I'm having the same issue as well. I use it in another project and everything works fine. Newest version I just downloaded doesnt work. 

Running a blank Laravel project, only package I have loaded at this point is this one, 
If I do this: $data = fastexcel()->import(storage_path($path));  it returns []
If I do this: $sheets = (new FastExcel)->importSheets(storage_path($path)); it returns [[]]

The file is very small as it's a test file I just have 1 sheet that has a value in a1 and b1.
